client/src/components/NewBlockForm.tsx

```diff
@@ -87,6 +100,9 @@ export class NewBlockFormPresentational extends React.Component<any, any> {
             : 1,
         }}
       >
+        {this.state.shouldRedirectToBreakPage && (
+          <Redirect to={`/break?e=${this.props.experimentId}`} />
+        )}
```

Comment:
Haven't looked into the object lifecycle here, but noticing that `shouldRedirectToBreakPage` isn't set back to `false`. Is that okay?

Reply:
Yeah this should be fine. There are examples set up in exactly this way, and I assume `<Redirect />` unmounts the original component, resetting the state. It's unclear how you could even set it back to false.